css/global.css

```diff
@@ -0,0 +1,4 @@
+.ad-banner,
+div.feed-shared-update-v2:has(div):has(div.update-components-actor):has(.app-aware-link):has(div.update-components-actor__meta):has(span.update-components-actor__sub-description):has(span.visually-hidden):has(span[aria-hidden="true"]):has(span):not(:has([type="globe-americas"])){
```

Comment:
This uses the :has and :not selectors in tandem! 
This looks for a div with a class of `feed-shared-update-v2` (the main element that is used for all items in the newsfeed) that has a child div with a class of `update-components-actor` that has a child element with a class of `app-aware-link` that has a child div with a class of `update-components-actor__meta` that has a child span with a class of `update-components-actor__sub-description` that has a child span element with a class of `visually-hidden` that has a child span element with the `aria-hidden` attribute set to `true`, that has a child span element that does *not* have `type` set to `globe-americas`. This then hides the parent div with the class `feed-shared-update-v2` by removing it from the DOM if the above conditions were met. The HTML is the same for regular updates vs. sponsored posts except sponsored posts have a custom HTML element with `type="globe-americas"`.